x/msgfees/types/msgs.go

```diff
@@ -50,9 +55,29 @@ func (msg MsgAssessCustomMsgFeeRequest) ValidateBasic() error {
 	if !msg.Amount.IsPositive() {
 		return errors.New("amount must be greater than zero")
 	}
+	_, err = msg.GetBips()
+	if err != nil {
+		return err
+	}
 	return nil
 }
 
+// GetBips converts the msg RecipientBasisPoints to a uint32 basis point value 0 - 10,000
+func (msg MsgAssessCustomMsgFeeRequest) GetBips() (uint32, error) {
+	if msg.RecipientBasisPoints == "" {
+		return AssessCustomMsgFeeBips, nil
+	}
+	bips, err := strconv.ParseUint(msg.RecipientBasisPoints, 10, 32)
+	if err == nil {
+		if bips > 10_000 {
+			return 0, fmt.Errorf("recipient basis points can only be between 0 and 10,000 : %v", msg.RecipientBasisPoints)
+		}
+	}
+
+	return uint32(bips), err
+}
+
+// GetSignBytes encodes the message for signing
```

Comment:
This comment looks to be reversed with the comment on `GetSignerBytes` method in `ln:89`.
```suggestion
// GetSigners indicates that the message must have been signed by the address provided
```

Reply:
I copied and pasted this from how we have it for 5 the other modules (25 total times). I would rather keep it consistent with what we already have in our other modules.  So, I think we change all of them now or do it in some other PR?

Reply:
If it exists in other modules as well then we'll need to do it in a separate PR.

Reply:
I created an issue for this.